src/lib/stitching/lewitt/schema.ts

```diff
@@ -0,0 +1,37 @@
+import { createLewittLink } from "./link"
+import {
+  makeRemoteExecutableSchema,
+  transformSchema,
+  RenameTypes,
+} from "graphql-tools"
+import { readFileSync } from "fs"
+
+export const executableLewittSchema = async () => {
+  const lewittSDL = readFileSync("src/data/lewitt.graphql", "utf8")
+  const lewittLink = createLewittLink()
+
+  const schema = await makeRemoteExecutableSchema({
+    schema: lewittSDL,
+    link: lewittLink,
+  })
+
+  // Remap the names of certain types from Lewitt to fit in the larger
+  // metaphysics ecosystem.
+  const remap = {
+    Currencies: "PartnerInvoiceCurrencies",
+    Invoice: "PartnerInvoice",
+    ArtworkGroup: "PartnerInvoiceArtworkGroup",
+    LineItem: "PartnerInvoiceLineItem",
+    MerchantAccount: "PartnerMerchantAccount",
```

Comment:
iirc @oxaudo @ashkan18 and I had settled on even more specific naming, because a partner may have other types of invoices at some point, so something like `PartnerProduct*` to signal it’s about a product that the partner sells.

Reply:
I just changed the rename function to prefix all of the classes